Code/Editor/Plugins/EditorAssetImporter/AssetImporterWindow.cpp

```diff
@@ -50,7 +50,7 @@ const AZ::Uuid AssetImporterWindow::s_browseTag = AZ::Uuid::CreateString("{C240D
 void MakeUserFriendlySourceAssetPath(QString& out, const QString& sourcePath)
 {
     char devAssetsRoot[AZ_MAX_PATH_LEN] = { 0 };
-    if (!gEnv->pFileIO->ResolvePath("@devroot@", devAssetsRoot, AZ_MAX_PATH_LEN))
+    if (!gEnv->pFileIO->ResolvePath("@engroot@", devAssetsRoot, AZ_MAX_PATH_LEN))
     {
         out = sourcePath;
         return;
```

Comment:
What is this function even originally trying to do and who calls it?  I doubt the engineroot is actually correct here, right?  Source assets could come from anywhere, but chances are if you're importing it you want it into your project)/Assets folder (note, not project root

Reply:
Yeah this should be the Project Root folder and not the Engine Root folder.
Ideally this should use `@projectsourceassets@` alias, but in order for that to work all of the Assets would need to be moved into the `<project-root>/Assets` directory for each project.